hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestSimpleRequestController.java

```diff
@@ -358,10 +361,8 @@ public void testWaitForMaximumCurrentTasks() throws Exception {
       try {
         barrier.await();
         controller.waitForMaximumCurrentTasks(max.get(), 123, 1, null);
-      } catch (InterruptedIOException e) {
+      } catch (InterruptedIOException | InterruptedException | BrokenBarrierException e) {
```

Comment:
Just throw the exception out? The fail message will contain the message of the exception.

Reply:
The Java compiler doesn't like it because of the lambda.

Reply:
Ah, got it.